Sources/LocalReceiptParsing/ReceiptParsingError.swift

```diff
@@ -15,28 +15,28 @@
 
 import Foundation
 
-enum ReceiptReadingError: Error, Equatable {
+extension ReceiptParser {
 
-    case missingReceipt,
```

Comment:
This one was never used.

Reply:
I suppose that's because the parser takes in a data object, so it doesn't allow you to call it with no receipt. 
For the public API I think that's something that we could abstract away, right? 
Meaning the fetching and / or refreshing of the receipt, so that the API allows you to either send it receipt data or just get the default receipt from device. Or maybe just the latter

Reply:
Not sure what you mean.
As for why this was not used, it's because it's detected earlier and we create an `ErrorCode` instead.